### ISSUE TITLE
Loss function & #iterations

### ISSUE THREAD

**Issue by mmfrxx** [AlexeyAB/darknet#3357]
Hello. I cloned the repository of pjreddie and tried to train the model on my training set. Then I wanted to look at loss graph, however, when I added -map there was nothing. What should I change in pjreddie's code in order to get the view at loss function. Additionally, now my code makes 360 iterations, how can I increase number of iterations?
